drivers/SmartThings/zigbee-button/src/init.lua

```diff
@@ -65,6 +65,7 @@ end
 local function added_handler(self, device)
   device:emit_event(capabilities.button.supportedButtonValues({"pushed","held","double"}))
   device:emit_event(capabilities.button.numberOfButtons({value = 1}))
+  device:emit_event(capabilities.button.button({"pushed"}))
```

Comment:
Can you update the unit tests to check for this too?

Reply:
Also I got this from @greens 's PR, but this is slightly better because it doesn't generate a state change event: `device:emit_event(capabilities.button.button.pushed({state_change = false}))`. I already tested it and it worked fine with this change.